lib/services/role.go

```diff
@@ -1188,15 +1188,14 @@ func (set RoleSet) AdjustClientIdleTimeout(timeout time.Duration) time.Duration
 	return timeout
 }
 
-// AdjustDisconnectExpiredCert adjusts the value based on the role set
-// the most restrictive option will be picked
-func (set RoleSet) AdjustDisconnectExpiredCert(disconnect bool) bool {
+// DisconnectExpiredCert returns the most restrictive DisconnectExpiredCert options from the role set.
+func (set RoleSet) DisconnectExpiredCert() bool {
 	for _, role := range set {
 		if role.GetOptions().DisconnectExpiredCert.Value() {
-			disconnect = true
+			return true
 		}
 	}
-	return disconnect
+	return false
```

Comment:
I don't think this is equivalent change. Previously, if `disconnect` variable passed here was `true` (i.e. cluster level setting was `true`), we would return true here, now we return false.

Reply:
Fixed